manifests/server/service.pp

```diff
@@ -9,7 +10,7 @@
     }
   }
 
-  file { $mysql::params::log_error:
```

Comment:
I believe so, yes. Otherwise we would have seen compile errors because of undef titles :). I could surround it with an `if` but I don't know if not having an error log is a valid use case.